app/src/main/java/com/example/stretchy/features/traininglist/ui/composable/TrainingListComposable.kt

```diff
@@ -319,6 +385,39 @@ private fun TrainingComposable(
     )
 }
 
+private fun getDataFromResult(
+    result: androidx.activity.result.ActivityResult,
+    context: Context,
+): String? {
+    val data: Uri? = result.data?.data
+    if (data != null && getFileName(
+            data,
+            context
+        ).contains(DataExporterImpl.dataTransportFileExt)
+    ) {
```

Comment:
formatting. 
This looks more clear
![image](https://github.com/valis-organization/stretchy/assets/29818981/9552f863-6775-40bf-837d-d956a457cfcd)